packages/react-components/react-radio/src/components/Radio/useRadioStyles.ts

```diff
@@ -116,6 +115,15 @@ const useInputStyles = makeStyles({
       },
     },
   },
+
+  after: {
+    height: '100%',
+    width: `calc(${indicatorSize} + 2 * ${tokens.spacingHorizontalS})`,
+  },
+  below: {
+    height: `calc(${indicatorSize} + ${tokens.spacingVerticalS})`,
+    width: '100%',
+  },
```

Comment:
The `base` styles specify width and height as 100%, so you only need to override the appropriate one:
```suggestion
  after: {
    width: `calc(${indicatorSize} + 2 * ${tokens.spacingHorizontalS})`,
  },
  below: {
    height: `calc(${indicatorSize} + 2 * ${tokens.spacingVerticalS})`,
  },
```

Reply:
You're right, removed.